app/src/main/java/com/willowtree/vocable/settings/EditKeyboardFragment.kt

```diff
@@ -273,11 +273,7 @@ class EditKeyboardFragment : BaseFragment() {
                 if (isCategory) {
                     parentFragmentManager.popBackStack()
                 } else {
-                    parentFragmentManager
-                        .beginTransaction()
-                        .replace(R.id.settings_fragment_container, EditPresetsFragment())
-                        .addToBackStack(null)
-                        .commit()
+                    parentFragmentManager.popBackStack()
```

Comment:
You should be able to get rid of the if-statement now if they're both doing the same thing